lib/database/instance-memory.json

```diff
@@ -0,0 +1,525 @@
+  "c6id.2xlarge": 16384,
+  "c6id.12xlarge": 98304,
+  "m6a.32xlarge": 524288,
+  "r5dn.4xlarge": 131072,
+  "t2.small": 2048,
+  "m6a.xlarge": 16384,
+  "g3s.xlarge": 31232,
+  "r5d.metal": 786432,
+  "im4gn.4xlarge": 65536,
+  "r5ad.4xlarge": 131072,
+  "r6g.xlarge": 32768,
+  "r4.large": 15616,
+  "i3en.2xlarge": 65536,
+  "g5.8xlarge": 131072,
+  "r5b.8xlarge": 262144,
+  "i3.16xlarge": 499712,
+  "d3.4xlarge": 131072,
+  "r6a.large": 16384,
+  "m6i.xlarge": 16384,
+  "r5.12xlarge": 393216,
+  "r5a.8xlarge": 262144,
+  "dl1.24xlarge": 786432,
+  "im4gn.large": 8192,
+  "p3.8xlarge": 249856,
+  "i3en.large": 16384,
+  "t3a.nano": 512,
+  "c6i.metal": 262144,
+  "i3en.xlarge": 32768,
+  "c6g.4xlarge": 32768,
+  "m5a.large": 8192,
+  "r6gd.12xlarge": 393216,
+  "m6id.4xlarge": 65536,
+  "p2.16xlarge": 749568,
+  "t2.xlarge": 16384,
+  "r6a.16xlarge": 524288,
+  "r5dn.2xlarge": 65536,
+  "r5n.8xlarge": 262144,
+  "r5a.24xlarge": 786432,
+  "i4i.xlarge": 32768,
+  "c7g.12xlarge": 98304,
+  "d2.4xlarge": 124928,
+  "m6a.24xlarge": 393216,
+  "p2.xlarge": 62464,
+  "is4gen.medium": 6144,
+  "t3.small": 2048,
+  "p3.16xlarge": 499712,
+  "m6id.8xlarge": 131072,
+  "im4gn.16xlarge": 262144,
+  "c5ad.8xlarge": 65536,
+  "x2iezn.6xlarge": 786432,
+  "r6i.8xlarge": 262144,
+  "c6gd.medium": 2048,
+  "vt1.24xlarge": 196608,
+  "r6g.metal": 524288,
+  "g4ad.16xlarge": 262144,
+  "m4.10xlarge": 163840,
+  "m5.large": 8192,
+  "z1d.6xlarge": 196608,
+  "x2iedn.24xlarge": 3145728,
+  "x1e.xlarge": 124928,
+  "m6gd.2xlarge": 32768,
+  "z1d.large": 16384,
+  "x1.32xlarge": 1998848,
+  "m6a.48xlarge": 786432,
+  "g5g.xlarge": 8192,
+  "u-18tb1.112xlarge": 18874368,
+  "g4dn.4xlarge": 65536,
+  "c5a.large": 4096,
+  "g5.48xlarge": 786432,
+  "c6id.4xlarge": 32768,
+  "c6a.large": 4096,
+  "c5d.9xlarge": 73728,
+  "r6i.large": 16384,
+  "c6gd.12xlarge": 98304,
+  "c6gd.large": 4096,
+  "u-9tb1.112xlarge": 9437184,
+  "t3a.micro": 1024,
+  "m6a.8xlarge": 131072,
+  "d3en.12xlarge": 196608,
+  "m5a.12xlarge": 196608,
+  "c5n.xlarge": 10752,
+  "m5d.4xlarge": 65536,
+  "p3dn.24xlarge": 786432,
+  "i3en.6xlarge": 196608,
+  "m6i.2xlarge": 32768,
+  "i3.4xlarge": 124928,
+  "is4gen.8xlarge": 196608,
+  "x2iedn.8xlarge": 1048576,
+  "t2.nano": 512,
+  "c5.18xlarge": 147456,
+  "r4.4xlarge": 124928,
+  "r6i.2xlarge": 65536,
+  "f1.16xlarge": 999424,
+  "m5a.xlarge": 16384,
+  "m6gd.large": 8192,
+  "t3a.xlarge": 16384,
+  "i3en.metal": 786432,
+  "g4dn.2xlarge": 32768,
+  "d3en.xlarge": 16384,
+  "r5dn.24xlarge": 786432,
+  "c6gd.8xlarge": 65536,
+  "m5a.24xlarge": 393216,
+  "i4i.8xlarge": 262144,
+  "m6id.large": 8192,
+  "z1d.3xlarge": 98304,
+  "c6g.metal": 131072,
+  "r5n.large": 16384,
+  "f1.4xlarge": 249856,
+  "d2.xlarge": 31232,
+  "m6i.12xlarge": 196608,
+  "r5n.metal": 786432,
+  "m6a.12xlarge": 196608,
+  "g4dn.8xlarge": 131072,
+  "m6g.2xlarge": 32768,
+  "g4ad.xlarge": 16384,
+  "m6g.medium": 4096,
+  "c6g.2xlarge": 16384,
+  "g5g.4xlarge": 32768
+}
```

Comment:
Good lord, I hope you didn't type all these in by hand :-)

Reply:
These are just the ones that I had committed to memory ;)